### ISSUE TITLE
Error with moon graphic

### ISSUE THREAD

**Issue by jvillasante** [cadadr/forecast.el#9]
I'm getting this error:

```
error in process filter: forecast--insert-sun-moon-graphic: Args out of range: ">—————————————————————————————————————————————————————————————————————————————————————————————————————————————————————<", 121
error in process filter: Args out of range: ">—————————————————————————————————————————————————————————————————————————————————————————————————————————————————————<", 121
```

I'm using Source Code Pro as font.
Ultimately I don't really need the moon forecast, can it be an option to take it out of the forecast output?

**Comment by cadadr** [comment]
Hi, thanks for the report. The problem is related to the sun graphic, not the moon graphic, though I'll consider adding in a switch for it.  Could you report me which version of forecast.el you're using?  If you can open the file, about the 7th line you should see a line like this:

    ;; Version: 0.3.0

please send me that one.  I haven't been able to reproduce the problem on my version.

**Comment by cadadr** [comment]
This error was fixed in commit ce511ddb921.  The issue was miscalculation of an index that was dependent on current time.  Late in the day the index overflew the array.  The issue does not have anything to do with the moon graphic.  Thanks for the report again.
